next-cloudinary/src/plugins/cropping.js

```diff
@@ -8,21 +8,21 @@ export function plugin({ cldImage, options, cldOptions } = {}) {
 
   let transformationString = `c_${crop},w_${width}`;
 
-  if ( !gravity && cropsGravityAuto.includes(crop) ) {
+  if ( !gravity ) {
+    if ( cropsGravityAuto.includes(crop) && !gravity ) {
```

Comment:
you can probably get rid of the additional `!gravity` check here as at this point, we're already in an `if` clause where the case is that we dont have gravity set

Reply:
i was just suggesting you could instead do this:
```
  if ( !gravity ) {
    if ( cropsGravityAuto.includes(crop)) {
```

as the additional `!gravity` on the second line is redundant :) but not a big deal

Reply:
That makes sense!! I will change it!